theme/theme.go

```diff
@@ -142,27 +147,27 @@ func (t *builtinTheme) initFonts() {
 	}
 }
 
-// TextFont returns the font path for the regular font style
```

Comment:
I just noticed that this should be "resource" not "path", can you adjust your fix? (same for the rest)

Reply:
ahhhh, the search and replace when I was editing the SVG must have caught these up in the change! sorry about that.